src/main/webapp/dev/js/main/adminSearch.es6

```diff
@@ -144,4 +145,54 @@ $(document).ready(() => {
     $('#btn-collapse-all-students').on('click', () => {
         adminSearchCollapseAllStudents();
     });
+
+    $('.openEmailApplicationDefaultValues').submit((e) => {
+        e.preventDefault();
+        const entry = e.target;
+        const studentEmail = $(entry).parentsUntil('.fslink_student').find('input[name=studentEmail]').val();
+        const subjectType = $(entry).find('input[name=subjectType]').val();
+        const courseName = $(entry).find('input[name=courseName]').val();
+        const courseId = $(entry).find('input[name=courseId]').val();
+        const studentName = $(entry).find('input[name=studentName]').val();
+        const sessionStatus = $(entry).find('input[name=sessionStatus]').val();
+        const feedbackSessionName = $(entry).find('#feedback-session-name').text();
+
+        const relatedLink = encodeURIComponent($(entry).find('input[name=relatedLink]').val());
+        const googleSignup = encodeURIComponent('https://accounts.google.com/NewAccount');
+        const supportEmail = `${ParamsNames.TEAMMATES_EMAIL_ADDRESS}`;
+
+        const bodyHeader = `Hello ${studentName},`;
+        const bodyFooter = '%0D%0A%0D%0AIf you encounter any problems when using the system,'
+                         + ` you can email TEAMMATES support team at ${supportEmail}.`
+                         + '%0D%0A%0D%0ARegards,%0D%0ATEAMMATES Team.';
+
+        const uniqueLinkMessage = 'The above link is unique to you. Please do not share it with your classmates.';
+
+        let subject = '';
+        let bodyContent = '';
+
+        if (subjectType === 'Invitation to join course') {
+            subject = `TEAMMATES: ${subjectType} [${courseName}][Course ID: ${courseId}]`;
+            bodyContent =
+                    `%0D%0A%0D%0AThe course ${courseName} is using the TEAMMATES System to collect feedback.`
+                    + ` %0D%0A%0D%0ATo "join" the course, please go to this Web address: ${relatedLink}`
+                    + ' %0D%0A%0D%0A*If prompted to log in, use your Google account to log in. If you do not have'
+                    + ` a Google account, please create one from the Google Accounts page: ${googleSignup}`
+                    + ` %0D%0A*${uniqueLinkMessage} %0D%0A%0D%0ANote that If you wish to access TEAMMATES`
+                    + ' without using your Google account, you do not need to ‘join’ the course as instructed above.'
+                    + ' You will still be able to submit/view feedback by following the instructions sent to you by'
+                    + ' TEAMMATES at the appropriate times. However, we recommend joining the course using your Google'
+                    + ' account, because it gives you more convenient access to all your feedback stored in TEAMMATES.';
+        } else {
+            subject = `TEAMMATES: ${subjectType} [Course: ${courseName}][Feedback Session: ${courseId}]`;
```

Comment:
@chowyb generally the feedback session name is this big 
`Second team feedback session [Sun, 01 Apr 2012, 11:59 PM UTC+0800 - Thu, 04 Jan 2018, 11:59 PM UTC+0800]`.

If I add the Feedback session name in the subject, don't you think the subject will then be too big or should I skip this part `[Sun, 01 Apr 2012, 11:59 PM UTC+0800 - Thu, 04 Jan 2018, 11:59 PM UTC+0800]` and keep this only `Second team feedback session`.